backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/TargetLawServiceTest.java

```diff
@@ -62,4 +65,28 @@ void canLoadTargetLawXmlByEli() {
     verify(loadTargetLawXmlAdapter, times(1))
         .loadTargetLawXmlByEli(argThat(command -> command.eli().equals(eli)));
   }
+
+  @Test
+  void appliesTimeMachineWhenCalled() {
```

Comment:
I'm a little confused as to what's tested, here.

Is it that `timeMachine.apply()` has been called? If yes, then maybe rather `callsTimeMachineApply()`.